src/Raven.Server/Web/System/BuildVersionHandler.cs

```diff
@@ -45,6 +49,8 @@ private static byte[] GetVersionBuffer()
         public async Task Get()
         {
             HttpContext.Response.Headers.Add(Constants.Headers.ServerStartupTime, ServerStore.Server.Statistics.StartUpTime.GetDefaultRavenFormat(isUtc: true));
+            HttpContext.Response.Headers.Add(Constants.Headers.IsLinux, IsLinux);
```

Comment:
You can use a boolean here, `bool.ToString()` is cached.

Reply:
done